src/Schema.php

```diff
@@ -1095,6 +1114,24 @@ protected function validateEnum($value, ValidationField $field) {
     }
 
     /**
+     * Call all of the filters attached to a field.
+     *
+     * @param mixed $value The field value being filtered.
+     * @param ValidationField $field The validation object.
+     * @return mixed Returns the filtered value. If there are no filters for the field then the original value is returned.
+     */
+    protected function callFilters($value, ValidationField $field) {
+        // Strip array references in the name except for the last one.
+        $key = preg_replace(['`\[\d+\]$`', '`\[\d+\]`'], ['[]', ''], $field->getName());
```

Comment:
We should probably stick to `//` for regexes when it does not imply that we will need to escape the slashes from the content.